lib/atlas/preset/scaling.rb

```diff
@@ -17,6 +17,10 @@ class Scaling
       validates :area_attribute, presence: true
       validates :value,          presence: true, numericality: true
       validates :base_value,     presence: true, numericality: true
+
+      def factor
+        value.to_r / base_value.to_r
```

Comment:
I alreadt removed, the `scaling_factor` in the `Scaler` [here](https://github.com/quintel/atlas/pull/98/files#diff-570156e24823180b9e2c105aa84dcaf4L40) though.